packages/adminrouter/extra/src/lib/auth/open.lua

```diff
@@ -195,6 +195,11 @@ function _M.init(use_auth)
         return res.do_authn_and_authz_or_exit()
     end
 
+    -- /dcos-ui-service/v1/
+    res.access_dcos_ui_service_endpoint = function()
+        return res.do_authn_and_authz_or_exit("dcos:superuser");
```

Comment:
What's the idea behind that?

The UI team went through great lengths in the previous to make the UI work not only for superusers.

Reply:
UI will still work, it's just updating UI, so calling this api will be available to superusers only.

Reply:
Ha! :) As you see I understood nothing about this PR so far.

Reply:
Changed the name to reflect the intent better